spec/models/quiz_spec.rb

```diff
@@ -5,11 +5,12 @@
   it { is_expected.to have_db_index(:course_id) }
   it { is_expected.to validate_presence_of(:title) }
   it { is_expected.to validate_presence_of(:description) }
+  it { is_expected.to validate_presence_of(:questions).
```

Comment:
Avoid using {...} for multi-line blocks.<br>Block body expression is on the same line as the block start.